extensions/stock-dashboard-python/requirements.txt

```diff
@@ -0,0 +1,4 @@
+dash==2.16.0
+dash_bootstrap_components==1.5.0
+pandas==2.2.1
+plotly==5.24.1
```

Comment:
We don't have to do this today, but we should consider using constraints here rather than strict pins.